From c33339a87ae7e194b22441b08d918e354c53ad03 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Wed, 27 Jun 2018 18:04:54 +0800
Subject: [PATCH 1323/1345] arch: arm64: dts: add UART nodes for cp110-maser

Add the missing UART nodes.
This patch refer to sdk kernel commit ID
8735edf663a5("fix: dts: add UART nodes for CP-110")

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/marvell/armada-cp110-master.dtsi      |   40 ++++++++++++++++++++
 1 file changed, 40 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
index c245a64..8e2b7a8 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
@@ -396,6 +396,46 @@
 				nand-ecc-step-size = <512>;
 				status = "disabled";
 			};
+
+			serial@702000 {
+				compatible = "snps,dw-apb-uart";
+				reg = <0x702000 0x100>;
+				reg-shift = <2>;
+				interrupts = <ICU_GRP_NSR 122 IRQ_TYPE_LEVEL_HIGH>;
+				reg-io-width = <1>;
+				clocks = <&cpm_syscon0 0  3>;
+				status = "disabled";
+			};
+
+			serial@702100 {
+				compatible = "snps,dw-apb-uart";
+				reg = <0x702100 0x100>;
+				reg-shift = <2>;
+				interrupts = <ICU_GRP_NSR 123 IRQ_TYPE_LEVEL_HIGH>;
+				reg-io-width = <1>;
+				clocks = <&cpm_syscon0 0  3>;
+				status = "disabled";
+			};
+
+			serial@702200 {
+				compatible = "snps,dw-apb-uart";
+				reg = <0x702200 0x100>;
+				reg-shift = <2>;
+				interrupts = <ICU_GRP_NSR 124 IRQ_TYPE_LEVEL_HIGH>;
+				reg-io-width = <1>;
+				clocks = <&cpm_syscon0 0  3>;
+				status = "disabled";
+			};
+
+			serial@702300 {
+				compatible = "snps,dw-apb-uart";
+				reg = <0x702300 0x100>;
+				reg-shift = <2>;
+				interrupts = <ICU_GRP_NSR 125 IRQ_TYPE_LEVEL_HIGH>;
+				reg-io-width = <1>;
+				clocks = <&cpm_syscon0 0  3>;
+				status = "disabled";
+			};
 		};
 
 		cpm_pcie0: pcie@f2600000 {
-- 
1.7.9.5

